src/event_key.rs

```diff
@@ -38,8 +39,7 @@ impl EventKey {
         self.as_ref().group
     }
 
-    // TODO: to be uncommented when the field will be generated
-    // pub fn get_is_modifier(&self) -> bool {
-    //     from_glib(self.as_ref().is_modifier)
-    // }
+    pub fn get_is_modifier(&self) -> bool {
+        self.as_ref().is_modifier != 0
```

Comment:
Its bitfield, so IMHO better something like `self.as_ref().is_modifier & 1 != 0`,
same for `is_stop`